src/include/sof/drivers/ssp.h

```diff
@@ -221,7 +221,7 @@ extern const struct dai_driver ssp_driver;
 #define SSP_MAX_SEND_TIME_PER_SAMPLE 125
 
 /* SSP flush timeout in microseconds */
-#define SSP_RX_FLUSH_TIMEOUT	200
+#define SSP_RX_FLUSH_TIMEOUT	500
```

Comment:
Aren't we using rate here ?

Reply:
> Aren't we using rate here ?

I wanted to use it as the total timeout control, now I change my mind to use something like this:
```
#define  SSP_RX_FLUSH_SAMPLE_MAX 16
```

Will update it in the PR.